pom.xml

```diff
@@ -758,6 +753,7 @@
         </profile>
     </profiles>
     <modules>
+        <module>cloudslang-content-dependencies</module>
```

Comment:
IMHO build order is not guaranteed - one solution would be to add an unused declared dependency in the modules that depend on this one